package.json

```diff
@@ -22,6 +22,7 @@
     "mocha": "^10.2.0",
     "prettier": "^2.8.8",
     "sinon": "^18.0.0",
+    "svgo": "^3.3.2",
```

Comment:
i had no idea the website was just a prettified UI around this

Reply:
Yeah! Also this gave me the idea of locking the version, so that files are not modified often because of changes between version